test/e2e/README.md

```diff
@@ -0,0 +1,105 @@
+# Use E2E to test itself
+
+After updating the features of e2e, you can use the files in the test/e2e/ directory to perform testing for both new and old features of e2e.
+You can perform testing locally. And when you submit a pull request (PR), GitHub Actions will automatically run the tests.
+
+## Files introduction
+- `test/e2e`
+    - `concurrency`
+        - `fail-fast` (concurrency & fail-fast mode)
+          - `internal`
+            - `expected.yaml` 
+            - `verify.yaml` (configuration file for the inner infra E2E(concurrency & fail-fast))
+          - `expected.yaml`
+        - `non-fail-fast` (concurrency & non-fail-fast mode)
+          - `internal`
+              - `expected.yaml`
+              - `verify.yaml` (configuration file for the inner infra E2E(concurrency & non-fail-fast))
+          - `expected.yaml`
+    - `non-concurrency` 
+        - `fail-fast` (non-concurrency & fail-fast mode)
+            - `internal`
+              - `expected.yaml`
+              - `verify.yaml` (configuration file for the inner infra E2E(non-concurrency & fail-fast))
+            - `expected.yaml`
+        - `non-fail-fast` (non-concurrency & non-fail-fast mode)
+            - `internal`
+                - `expected.yaml`
+                - `verify.yaml` (configuration file for the inner infra E2E(non-concurrency & non-fail-fast))
+            - `expected.yaml`
+    - `docker-compose.yaml` (run a httpbin container, which can return YAML data)
+    - `e2e.yaml` (configuration file for the outer infra E2E)
+
+## Tips
```

Comment:
One detailed example is enough, the purpose of the readme is to tell new contributors how to add new cases for their changes.